core/quantity/src/Formatter/Formatter.ts

```diff
@@ -355,9 +357,24 @@ export class Formatter {
         formattedValue = stationString + fractionString;
       }
     }
+
+    if(spec.format.minWidth) {
+      formattedValue = this.countAndPad(formattedValue, spec.format.minWidth);
+    }
+
     return formattedValue;
   }
 
+  private static countAndPad(value: string, minWidth: number): string {
+    const regex = /[\d,.]/g;
```

Comment:
what if a user specified a different thousand and decimal separator? would the regex catch this?

Reply:
maybe having a separate unit test in NumericFormat.test.ts where a min width is specified and used, alongside a custom decimal separator would be good

Reply:
Good point. I copied this from the C++ implementation. I think you're right. I don't expect anybody is really using a different separator than comma and dot so this had gone unnoticed. Could you guys revisit this later at some point? It would have to be adjusted in native as well.